fern/definition/requests.yaml

```diff
@@ -0,0 +1,46 @@
+imports:
+  routes: ./routes.yml
+
+types:
+  HttpMethod:
+    enum:
+      - GET
+      - POST
+      - PUT
+      - DELETE
+      - PATCH
+
+  RequestReport:
+    properties:
+      baseUrl: string
+      path: string
+      method: HttpMethod
+      pathParams: optional<map<string, string>>
+      queryParams: optional<map<string, string>>
+      headerParams: optional<map<string, string>>
+      bodyParams: optional<string>  
+      formParams: optional<map<string, string>>  
+      multipartParams: optional<map<string, string>>  
+      vulnTypes: optional<list<string>>
```

Comment:
These are the vuln types we were checking, right? I think this probably deserves a docs string and should maybe think about `VulnType` being an enum